itests/testdata/transfer_tx_testdata.go

```diff
@@ -49,38 +71,389 @@ type CommonTransferData struct {
+			&assetId,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(assetAmount/8),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			nil,
+			TransferExpectedValuesPositive{
+				WavesDiffBalanceSender:    utl.MinTxFeeWaves,
+				AssetDiffBalance:          assetAmount / 8,
+				WavesDiffBalanceRecipient: 0,
+			},
+		),
+		"Alias as string": *NewTransferTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			rcpntAlias,
+			nil,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(wavesAmount/8),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
 			nil,
 			TransferExpectedValuesPositive{
-				WavesDiffBalance: 100000,
-				AssetDiffBalance: 1,
+				WavesDiffBalanceSender:    utl.MinTxFeeWaves + wavesAmount/8,
+				AssetDiffBalance:          0,
+				WavesDiffBalanceRecipient: wavesAmount / 8,
 			},
 		),
 	}
+	return t
+}
+
+func GetTransferMaxAmountPositive(suite *f.BaseSuite, assetId crypto.Digest, accNumber int) map[string]TransferTestData[TransferExpectedValuesPositive] {
+	wavesAmount := utl.GetAvailableBalanceInWavesGo(suite, utl.GetAccount(suite, accNumber).Address)
+	var t = map[string]TransferTestData[TransferExpectedValuesPositive]{
+		"Max values for amount, attachment": *NewTransferTestData(
+			utl.GetAccount(suite, accNumber),
+			proto.NewRecipientFromAddress(utl.GetAccount(suite, utl.DefaultRecipientNotMiner).Address),
+			&assetId,
+			nil,
+			uint64(wavesAmount),
+			utl.MaxAmount,
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			proto.Attachment("2GjX8YcCcSdmYm3pVP41e1TL1t5nQrHUkCx6V9L7SC5LxQSEmcE3irKh2NtV2x57fNU5MoRML6CVyKVbatfbcNWst"+
+				"N3cuernPDaF4kgpn5g1DdPkfH6gge94TesYMkhdSHwVoVvXwacr"),
+			TransferExpectedValuesPositive{
+				WavesDiffBalanceSender:    wavesAmount,
+				AssetDiffBalance:          utl.MaxAmount,
+				WavesDiffBalanceRecipient: 0,
+			},
+		),
+	}
+	return t
+}
+
+func GetTransferNegativeData(suite *f.BaseSuite, assetId crypto.Digest) map[string]TransferTestData[TransferExpectedValuesNegative] {
+	assetAmount := utl.GetAssetBalanceGo(suite, utl.GetAccount(suite, utl.DefaultSenderNotMiner).Address, assetId)
+	invalidAssetId := utl.RandDigest(suite.T(), 32, utl.LettersAndDigits)
+	var t = map[string]TransferTestData[TransferExpectedValuesNegative]{
+		"Attachment > max": *NewTransferTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			proto.NewRecipientFromAddress(utl.GetAccount(suite, utl.DefaultRecipientNotMiner).Address),
+			&assetId,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(assetAmount),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			proto.Attachment("2GjX8YcCcSdmYm3pVP41e1TL1t5nQrHUkCx6V9L7SC5LxQSEmcE3irKh2NtV2x57fNU5MoRML6CVyKVbatfbcNWst"+
+				"N3cuernPDaF4kgpn5g1DdPkfH6gge94TesYMkhdSHwVoVvXwacr2Gj"),
+			TransferExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    errBrdCstMsg,
+				ErrBrdCstScalaMsg: "exceeds maximum length of 192",
```

Comment:
Wrong error message, according to NODE-2493, attachment size should compared for decoded base58 string. Max attachment size is 140

Reply:
Fix